examples/tensorflow-multigpu/README.md

```diff
@@ -0,0 +1,14 @@
+# TensorFlow 
+## Note: This example only runs on Pro and Enterprise
+
+This resource gives you example for running TensorFlow training tasks on multiple GPU inside Saturn Cloud. 
+
+[multi-gpu-tensorflow](multi-gpu-tensorflow.ipynb) train your model with TensorFlow on multiple GPUs. 
```

Comment:
```suggestion
This is an example for running TensorFlow training tasks on multiple GPUs inside of Saturn Cloud. It uses a single large resource with many GPUs attach to concurrently train a model. The example is contained in [multi-gpu-tensorflow.ipynb](multi-gpu-tensorflow.ipynb).

To train a model with a single GPU in TensorFlow, see the single GPU TensorFlow example on the Saturn Cloud docs.
```

I didn't link to the single GPU example here, but we'll want to update it once the docs are updated